src/site/content/en/blog/how-to-report-metrics/index.md

```diff
@@ -2,17 +2,17 @@
 title: How to report metrics and build a performance culture
 subhead: Understand why performance matters when it comes to conversion
 authors: 
-  - martinschierle
+ —martinschierle
```

Comment:
I think it's ok to leave the spaces in. That matches the other posts on the site.

Reply:
Oops. I'll fix it.